module/core/error_tools/Cargo.toml

```diff
@@ -17,6 +17,9 @@ Basic exceptions handling mechanism.
 categories = [ "algorithms", "development-tools" ]
 keywords = [ "fundamental", "general-purpose" ]
 
+[package.metadata]
+stable_status = "experimental"
```

Comment:
don't make this changes in this PR
open separate PR for changes like that, please